.github/ISSUE_TEMPLATE/learn-feedback.yml

```diff
@@ -0,0 +1,68 @@
+name: learn.microsoft.com feedback control
```

Comment:
this name and description will show up on the new issue chooser right? (https://github.com/Azure/azure-sdk-for-js/issues/new/choose) 

Is there no way to hide it? It feels like it might be confusing for folks who are filing a new issue

Reply:
The name and description will show up. See https://github.com/Azure/azure-sdk-for-java/issues/new/choose for an example. I haven't found a way to hide the template, which is why obnoxious emojis are present in the description.

Reply:
Got it. Not sure how it decides the order, but maybe we can figure out how to name hack it to the bottom of the list and more out of the way? Seems like it might be tied to the file name alphabetical order so `z-learn-freedback.yml` or something?

Also could you tweak the description be a little more direct like "Do not use this template directly, instead click the feedback option on a learn.microsoft.com page."?